.buildkite/pipeline.package.mbp.yml

```diff
@@ -17,6 +17,13 @@ steps:
       provider: "gcp"
       image: "${IMAGE_UBUNTU_X86_64}"
       machineType: "c2-standard-16"
+    plugins:
+      # See https://github.com/elastic/oblt-infra/blob/main/conf/resources/repos/fleet-server/01-gcp-buildkite-oidc.tf
+      # This plugin authenticates to Google Cloud using the OIDC token.
+      - elastic/oblt-google-auth#v1.2.0:
+          lifetime: 10800 # seconds
+          project-id: "elastic-observability-ci"
+          project-number: "911195782929"
```

Comment:
This section is repeated several times.  Would it be possible to use [references](https://quickref.me/yaml.html#reference) to avoid the repetition?

Reply:
I'm not a big fan of the references (it's a swiss knife 🔪 ). However I see https://buildkite.com/docs/pipelines/integrations/plugins/using#using-yaml-anchors-with-plugins

so let me change it

Reply:
See 8802df2c